sdk/go.mod

```diff
@@ -5,11 +5,14 @@ go 1.15
 replace go.opentelemetry.io/otel => ../
 
 require (
+	github.com/gogo/protobuf v1.3.2
 	github.com/google/go-cmp v0.5.6
+	// TODO switch to https://github.com/jaegertracing/jaeger-idl
+	github.com/jaegertracing/jaeger v1.24.0
```

Comment:
Instead of importing [jaegertracing/jaeger](https://github.com/jaegertracing/jaeger) we should import [jaegertracing/jaeger-idl](https://github.com/jaegertracing/jaeger-idl). This is currently blowing up `go.sum`.
In fact, we only need [proto/api_v2/sampling.proto](https://github.com/jaegertracing/jaeger-idl/blob/master/proto/api_v2/sampling.proto).

I'm not sure how to do this though, help/feedback is welcome!

We should add it as git submodule:

```
[submodule "jaeger-idl"]
	path = sdk/trace/internal/jaeger-idl
	url = https://github.com/jaegertracing/jaeger-idl
```

But I'm not sure how to generate and import the proto as part of the build process.
- Do I add a target in Makefile?
- Should I check in the generated code or should people generate it themselves first?

fyi, the jaegertracing/protobuf image doesn't work on my machine (Apple M1). So that's also making testing more difficult... 😬

Reply:
This would need to live in a separate module to ensure that the increased dependency footprint only impacts applications that choose explicitly to include this sampler.  Indeed, it might be best for this to live in the contrib repo as it is not required to conform to the spec.

Reply:
Sure, sounds good :+1: There aren't any other samplers yet in contrib, should I create a new folder, i.e. `samplers/jaeger_remote`?

Reply:
> Sure, sounds good  There aren't any other samplers yet in contrib, should I create a new folder, i.e. `samplers/jaeger_remote`?

Yeah, please create a new sampler directory. We can discuss in the contrib PR, but I would prefer `samplers/jaeger`.